os/drivers/seclink/seclink_drv_key.c

```diff
@@ -29,71 +29,80 @@
 #endif
 #define SLDRV_TAG "[SECLINK_DRV_KEY]"
 
-#define NKEY_SLOT 4
+#define NKEY_SLOT 64
+/*
+ * +-----------------+------------------+
+ * |    ROM (0~31)   | RAM(32~63)       |
+ * +-----------------+------------------+
+ */
+#define KEY_ROM_ENTRY 0
+#define KEY_RAM_ENTRY 32
 static uint8_t g_key_inuse[NKEY_SLOT] = {0,};
+#define KEYSLOT_USE 0
+#define KEYSLOT_NOTUSE 1
 
-static int _set_key_slot(uint8_t num)
+static inline int _is_rom_area(uint8_t num)
```

Comment:
You had better define return value and aligning is needed for all these function.
Also this is a driver, some of description of function also needed.